app.js

```diff
@@ -1,15 +1,17 @@
 const express = require("express");
-const { getAllTopics } = require("./topics.controllers");
+const { getAllTopics, getApiInfo } = require("./controllers/topics.contollers");
 const app = express();
 
+app.get("/api", getApiInfo);
+
 app.get("/api/topics", getAllTopics);
 
 app.all("/*", (req, res, next) => {
   res.status(404).send({ msg: "wrong path" });
 });
 
-app.use((err, req, res, next) => {
-  if (err) res.status(500).send({ msg: "internal server error" });
-});
+// app.use((err, req, res, next) => {
```

Comment:
Why is this commented out? It's a helpful error handler and will probably serve as the final error handler in the file by the end of the project. I would also recommend console logging the error to see exactly what happened.